khmer/_khmermodule.cc

```diff
@@ -466,6 +466,20 @@ ReadParser_iter_reads(PyObject * self, PyObject * args )
     return PyObject_SelfIter( self );
 }
 
+static
+PyObject *
+ReadParser_get_num_reads(khmer_ReadParser_Object * me, PyObject * args)
+{
```

Comment:
Yup, my thoughts exactly. A pox on all your `.class`es.

Was unaware of `tp_members`, my `Python.h` knowledge is limited :)

Will investigate this.